app/views/characters/_editor.haml

```diff
@@ -1,19 +1,19 @@
 %tbody
   %tr
-    %th.sub Name
+    %th.sub= f.label :name
     %td{class: cycle('even', 'odd')}= f.text_field :name, placeholder: "Character Name", class: 'text'
   %tr
-    %th.sub Nickname
+    %th.sub= f.label :template_name, 'Nickname'
     %td{class: cycle('even', 'odd')}= f.text_field :template_name, placeholder: "Template Nickname", class: 'text'
   %tr
-    %th.sub Screenname
+    %th.sub= f.label :screenname
     %td{class: cycle('even', 'odd')}= f.text_field :screenname, placeholder: "Screen Name", class: 'text'
   %tr
-    %th.sub Setting
+    %th.sub= f.label :setting_ids, 'Setting'
     %td{class: cycle('even', 'odd')}= tag_select(@character, f, :settings)
   %tr
     - klass = cycle('even', 'odd')
-    %th.sub.vtop Template
+    %th.sub.vtop= f.label :template_id
```

Comment:
Does this need 'Template' specified?

Reply:
Nope! Rails generates 'Template' correctly from that.